mne_bids/mne_bids.py

```diff
@@ -686,7 +686,7 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         acquisition=acquisition, suffix='channels.tsv', prefix=data_path)
     if ext not in ['.fif', '.ds', '.vhdr', '.edf', '.bdf', '.set']:
         bids_raw_folder = bids_fname.split('.')[0]
-        bids_fname = op.join(bids_raw_folder, bids_fname)
+        bids_fname = '/'.join([bids_raw_folder, bids_fname])
```

Comment:
why this? `op.join` should be agnostic of operating system. That's its purpose

Reply:
yes, but that is the problem. On my computer when I generate the scans.tsv they look like this:
```
filename	acq_time
meg\sub-4_ses-1_task-restingstate_run-1_meg\sub-4_ses-1_task-restingstate_run-1_meg.con	2018-10-26T10:42:43
meg\sub-4_ses-1_task-optimumMMN_run-1_meg\sub-4_ses-1_task-optimumMMN_run-1_meg.con	2018-10-26T11:34:46
```
This is because `os.sep` is `\` on windows.
We need to force the separator to be `/` so that it is generated correctly on all OS's

Reply:
okay I understand. But can you do the string replacement of os.sep only when passing to _scans_tsv? Otherwise, if another person makes some changes in the lines between this one and the call to _scans_tsv, it could lead to unexpected results in the future.

Reply:
ok, I see what you mean. Good call!